spec/API_specification/manipulation_functions.md

```diff
@@ -19,7 +19,7 @@ Joins a sequence of arrays along an existing axis.
 
 #### Parameters
 
--   **arrays**: _Tuple\[ &lt;array&gt;, ... ]_
+-   **arrays**: _Union\[Tuple\[ &lt;array&gt;, ... ], List\[ &lt;array&gt; ] ]_
```

Comment:
Would it make more sense to use `Sequence`, rather than a `Union`?

Reply:
That was mentioned here https://github.com/numpy/numpy/pull/18585#discussion_r591754462. If we agree Sequence is better we can use that instead.

Reply:
I am fairly happy with requiring a tuple or list. `Sequence` is quite broad (e.g. is an array a sequence?), and I'm not that interested in supporting such flexibility for custom containers, it ends up hurting more than helping often.